source/index.ts

```diff
@@ -99,6 +99,20 @@ export default class PQueue<QueueType extends Queue<EnqueueOptionsType> = Priori
 		return this._pendingCount < this._concurrency;
 	}
 
+	get concurrency(): number {
+		return this._concurrency;
+	}
+
+	set concurrency(newConcurrency: number) {
+		if (!(typeof newConcurrency === 'number' && newConcurrency >= 1)) {
+			throw new TypeError(`Expected \`concurrency\` to be a number from 1 and up, got \`${newConcurrency}\` (${typeof newConcurrency})`);
+		}
+
+		this._concurrency = newConcurrency;
+
+		while (this.tryToStartAnother()) {} // eslint-disable-line no-empty
```

Comment:
This "while" thing is now done in multiple places. Would be good to move it to a method.